drivers/sensor/lis2mdl/lis2mdl.c

```diff
@@ -134,7 +94,7 @@ static void lis2mdl_channel_get_mag(struct device *dev,
 	}
 
 	for (i = ofs_start; i <= ofs_stop; i++) {
-		cval = lis2mdl->mag[i] * lis2mdl->mag_fs_sensitivity;
+		cval = lis2mdl->mag[i] * 1500;
```

Comment:
Is this change really related to stmemsc i/f?

Reply:
No, it is a code cleaning. The sensitivity does not change and it is always 1500 mgauss/LSB.
I would keep it in the same commit even if not strictly related to stmemsc. Do you agree?

Reply:
No, please move to a separate commit

Reply:
ok, done!